src/coreclr/jit/fgbasic.cpp

```diff
@@ -559,11 +557,17 @@ void Compiler::fgReplaceJumpTarget(BasicBlock* block, BasicBlock* newTarget, Bas
         case BBJ_ALWAYS:
         case BBJ_EHCATCHRET:
         case BBJ_EHFILTERRET:
-        case BBJ_LEAVE: // This function will be called before import, so we still have BBJ_LEAVE
+        case BBJ_LEAVE: // This function can be called before import, so we still have BBJ_LEAVE
```

Comment:
The changes in this function look fine, but also it seems weird BBJ_NONE/etc. case don't assert unreached.

Reply:
Let me fix this.

It's also a bit odd that even `block` has a reasonable jump kind this method can silently do nothing if `block` doesn't jump to `oldTarget`, but there are places we rely on this behavior right now.